net/mosquitto/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 
 name                mosquitto
-version             1.5.5
+version             1.5.6
 revision            1
```

Comment:
The `revision` line needs to be reset to 0 when upgrading the `version`.

Reply:
Deleted the revision rather than reset value to 0 in anticipation of @pmetzger request :). Thank you for the correction.